src/adapter/zboss/adapter/zbossAdapter.ts

```diff
@@ -187,124 +210,202 @@ export class ZBOSSAdapter extends Adapter {
         });
     }
 
-    public async supportsChangeChannel(): Promise<boolean> {
-        return false;
-    }
-
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     public async changeChannel(newChannel: number): Promise<void> {
-        throw new Error(`Channel change is not supported for 'zboss' yet`);
+        const clusterId = Zdo.ClusterId.NWK_UPDATE_REQUEST;
+        const zdoPayload = Zdo.Buffalo.buildRequest(this.hasZdoMessageOverhead, clusterId, [newChannel], 0xfe, undefined, undefined, undefined);
+
+        await this.sendZdo(ZSpec.BLANK_EUI64, ZSpec.BroadcastAddress.SLEEPY, clusterId, zdoPayload, true /* handled below */);
+        await Wait(12000);
     }
 
     public async setTransmitPower(value: number): Promise<void> {
         if (this.driver.isInitialized()) {
             return await this.queue.execute<void>(async () => {
-                await this.driver.setTXPower(value);
+                await this.driver.execCommand(CommandId.SET_TX_POWER, {txPower: value});
             });
         }
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     public async addInstallCode(ieeeAddress: string, key: Buffer): Promise<void> {
+        logger.error(() => `NOT SUPPORTED: sendZclFrameToGroup(${ieeeAddress},${key.toString('hex')}`, NS);
         throw new Error(`Install code is not supported for 'zboss' yet`);
     }
 
     public async permitJoin(seconds: number, networkAddress: number): Promise<void> {
         if (this.driver.isInitialized()) {
-            return await this.queue.execute<void>(async () => {
-                await this.driver.permitJoin(networkAddress, seconds);
-                if (!networkAddress) {
-                    // send broadcast permit
-                    await this.driver.permitJoin(0xfffc, seconds);
+            const clusterId = Zdo.ClusterId.PERMIT_JOINING_REQUEST;
+
+            if (networkAddress !== undefined) {
+                // TODO: works on networkAddress === 0 for 'coordinator-only'?
+                await this.driver.execCommand(CommandId.ZDO_PERMIT_JOINING_REQ, {nwk: networkAddress, duration: seconds, tcSignificance: 1});
```

Comment:
remove it, since the command.ZDO_PERMIT_JOINING_REQ was removed from the description